test-prow-e2e.sh

```diff
@@ -26,6 +40,11 @@ export BRIDGE_BASE_ADDRESS
 # Add htpasswd IDP
 oc apply -f ./frontend/integration-tests/data/htpasswd-secret.yaml
 oc patch oauths cluster --patch "$(cat ./frontend/integration-tests/data/patch-htpasswd.yaml)" --type=merge
+set +x
```

Comment:
I'm tempted to suggest we leave `set -x` instead of using `echo` and our own messages. It might be better for debugging problems. If the command fails, is the error suppressed with how this is written?

Reply:
Maybe it's better to use `set +x`, but we should make sure any errors running the CLI command are included in the log. We might need a max retries as well or this will poll forever if something goes wrong.

Reply:
Updated to use less verbose log messages:

**Typical run would produce:**
```
waiting for authentication operator to start Progressing 'test' idp...
authentication operator started Progressing 'test' idp (duration: 0h:0m:45s)
waiting for authentication operator to finish Progressing 'test' idp...
authentication operator finished Progressing 'test' idp (duration: 0h:2m:15s)
```
**Maximum retries reached:**
```
waiting for authentication operator to start Progressing 'test' idp...
authentication operator: maximum retries reached
```
**Error:**
```
waiting for authentication operator to start Progressing 'test' idp...
Error: unknown shorthand flag: 'y' in -y
See 'oc get --help' for usage.
```
Question is do we want to exit the script due to an error reading authentication operator status, or just trap it and report?